libraries/redcore/layouts/toolbar/button/version.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * @package     Redcore
+ * @subpackage  Layout
+ *
+ * @copyright   Copyright (C) 2008 - 2015 redCOMPONENT.com. All rights reserved.
+ * @license     GNU General Public License version 2 or later, see LICENSE.
+ */
+
+defined('JPATH_REDCORE') or die;
+
+$data = $displayData;
+
+if (!isset($data['button']))
+{
+	throw new InvalidArgumentException('The button is not passed to the layout "button.version".');
+}
+
+/** @var RToolbarButtonStandard $button */
+$button   = $data['button'];
+$isOption = $data['isOption'];
```

Comment:
this is not used, if this is set to true, your option should be in <li> please check other buttons